test/users.spec.js

```diff
@@ -32,6 +31,21 @@ describe('Authentication', () => {
         });
     });
 
+    it('should register a user with valid details', done => {
```

Comment:
Please sort out the duplicate test (is it actually supposed to be a _test_ or is it setup for something?)

Reply:
Ok. I will look into this

Reply:
sorted out